v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -132,6 +134,35 @@ def requires(self) -> ReferenceDataset:
             RAW_DATASET_PATH: 'https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/clinvar.vcf.gz',
         },
     },
+    ReferenceDataset.splice_ai: {
+        ENUMS: {
+            'splice_consequence': [
+                'Acceptor gain',
+                'Acceptor loss',
+                'Donor gain',
+                'Donor loss',
+                'No consequence',
+            ],
+        },
+        SKIP_ENUM_ANNOTATION: True,
```

Comment:
In the original script for writing the hail table we were actually mapping the index of an array to one of the the above strings. It seemed very silly to run that enum and then immediately transmute it back to the index, so I instead am just directly annotating the enumerated field and skipping the transmute step. 
I do still think its better behavior to have the enum transmute run by default for everything with enums, as I think this edge case will be rare